native_store.go

```diff
@@ -30,6 +31,7 @@ const (
 	remoteCredentialsPrefix       = "docker-credential-"
 	emptyUsername                 = "<token>"
 	errCredentialsNotFoundMessage = "credentials not found in native keychain"
+	errExeNotInPathMessage        = "exec: \"docker-credential-desktop.exe\": executable file not found in $PATH"
```

Comment:
Consider adding the following code to `executer.go`.
```go
if execErr, ok := err.(*exec.Error); ok && execErr.Err == exec.ErrNotFound {
	return nil, errors.New("credentials store is configured to `desktop` but Docker Desktop seems not running")
}
```

Reply:
Added this part.